src/Sulu/Component/Webspace/Tests/Unit/PortalInformationTest.php

```diff
@@ -24,15 +28,33 @@ class PortalInformationTest extends TestCase
      */
     private $portalInformation;
 
+    /**
+     * @var ObjectProphecy<Webspace>
+     */
+    private ObjectProphecy $webspace;
+
+    /**
+     * @var ObjectProphecy<Portal>
+     */
+    private ObjectProphecy $portal;
+
+    /**
+     * @var ObjectProphecy<Localization>
+     */
+    private ObjectProphecy $localization;
+
```

Comment:
These properties have been added, because they were implicit properties before which is deprecated in php.